src/main/java/core/basesyntax/Color.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public enum Color {
+    black,
+    gray,
+    white,
+    red,
+    orange,
+    yellow,
+    green,
+    blue,
+    violet
```

Comment:
In Enum all values must be written in upper case

Reply:
done